tests/ValidatorTest.php

```diff
@@ -169,7 +169,10 @@ public function requiredDataProvider(): array
             'sort' => [
                 new InRange(
                     ['asc', 'desc'],
-                    skipOnEmpty: static fn (mixed $value, bool $isAttributeMissing): bool => $isAttributeMissing || $value === ''
+                    skipOnEmpty: static fn (
+                        mixed $value,
+                        bool $isAttributeMissing
+                    ): bool => $isAttributeMissing || $value === ''
```

Comment:
```suggestion
                    skipOnEmpty: static function (mixed $value, bool $isAttributeMissing): bool {
                        return $isAttributeMissing || $value === '';
                    }
```

Reply:
(Optional, code style) The style before the changes is better for one-liners for my opinion.